.automation/update-actions-version.sh

```diff
@@ -29,7 +29,7 @@ GetReleaseVersion() {
   echo "Getting the latest Release version from GitHub ..."
 
   # Get the latest release on the Repository
-  GET_VERSION_CMD=$(echo "${RELEASE_NAME}" | grep -E -o "v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+" 2>&1)
+  GET_VERSION_CMD="$(echo \"${RELEASE_NAME}\" | grep -E -o \"v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+\" 2>&1)"
```

Comment:
```suggestion
  GET_VERSION_CMD="$(echo "${RELEASE_NAME}" | grep -E -o "v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+" 2>&1)"
```

You may not need to escape the double quotes in `$()`.

Reply:
<!-- Error while uploading 8aff252ec3453498598f83c502001670.mp4 -->